<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameEditorWindow</class>
 <widget class="QMainWindow" name="GameEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор SIGame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="topBar">
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="text">
         <string>Создать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="titleLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Название игры:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gameTitleEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollContent">
       <layout class="QVBoxLayout" name="roundsLayout">
        <item>
         <widget class="QPushButton" name="addRoundButton">
          <property name="text">
           <string>+ Добавить раунд</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
